Description: Mark a couple of pthread structures as opaque.
 These structures cause bindgen layout test failures on armel and armhf, and
 seem to be implementation details of the pthread library, so mark them as
 opaque.
Author: Peter Michael Green <plugwash@debian.org>

Last-Update: 2023-09-20

--- rust-xkbcommon-sys-1.4.1.orig/build.rs
+++ rust-xkbcommon-sys-1.4.1/build.rs
@@ -18,6 +18,8 @@ fn main() {
 		builder = builder.header("src/wrapper-x11.h");
 	}
 
+	builder = builder.opaque_type("__cancel_jmp_buf_tag");
+	builder = builder.opaque_type("__jmp_buf_tag");
   // Finish the builder and generate the bindings.
   builder
 		.generate()
